src/renderer/utils/createStorage.ts

```diff
@@ -22,16 +22,16 @@
 // Keeps window.localStorage state in external JSON-files.
 // Because app creates random port between restarts => storage session wiped out each time.
 import path from "path";
-import { app, remote } from "electron";
 import { comparer, observable, reaction, toJS, when } from "mobx";
 import fse from "fs-extra";
 import { StorageHelper } from "./storageHelper";
 import { ClusterStore } from "../../common/cluster-store";
 import logger from "../../main/logger";
-import { getHostedClusterId } from "../../common/utils";
+import { getHostedClusterId, getPath, noop } from "../../common/utils";
+import { isIntegrationTesting } from "../../common/vars";
 
 const storage = observable({
-  initialized: false,
+  initialized: isIntegrationTesting,
```

Comment:
This feels odd, we should not change logic while running integration tests.

Reply:
Fair enough, the changing of userData path is probably better.